src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -130,17 +130,17 @@
                 "UnnecessarySemicolonAfterTypeMemberDeclaration",
                 "UnnecessarySemicolonInTryWithResources",
                 "FinalLocalVariable",
-                "MissingSwitchDefault",
-                "UnnecessaryParentheses",
-                "MultipleStringLiterals",
-                "MultipleVariableDeclarations",
-                "AvoidDoubleBraceInitialization",
-                "ReturnCount",
-                "CovariantEquals",
-                "EqualsAvoidNull",
-                "HiddenField",
-                "NoClone",
-                "IllegalInstantiation"
+                "MissingSwitchDefault"
+                //                "UnnecessaryParentheses",
+                //                "MultipleStringLiterals",
+                //                "MultipleVariableDeclarations",
+                //                "AvoidDoubleBraceInitialization",
+                //                "ReturnCount",
+                //                "CovariantEquals",
+                //                "EqualsAvoidNull",
+                //                "HiddenField",
+                //                "NoClone",
+                //                "IllegalInstantiation"
```

Comment:
Please add more checks, may be all that remain

Reply:
I didn't understand what checks you are trying to refer. Do you refer the `i+1`th PR checks?